site/app/(home)/page.tsx

```diff
@@ -78,6 +61,29 @@ export default async function Home({ searchParams }: HomeProps) {
                     </details>
                   ))}
               </div>
+              <div className="border-ctx bg-charcoal text-putty-light border -m-0.5px font-mono text-sm leading-mono font-normal uppercase -mx-1px -mt-1px h-2 px-1 flex gap-0.25 items-center justify-center text-center">
+                Set theme colors <ThemeColorPopover />
+              </div>
+              <div className="border-ctx lg:border-b p-1 flex flex-col sm:flex-row justify-center gap-0.5">
+                <div className="flex items-center gap-0.25">
+                  <ColorPicker id="primary-color" defaultValue="#ffffff" />
+                  <label htmlFor="primary-color" className="uppercase text-sm font-mono">
+                    Primary
+                  </label>
+                </div>
+                <div className="flex items-center gap-0.25">
+                  <ColorPicker id="secondary-color" defaultValue="#ffffff" />
+                  <label htmlFor="secondary-color" className="uppercase text-sm font-mono">
+                    Secondary
+                  </label>
+                </div>
+                <div className="flex items-center gap-0.25">
+                  <ColorPicker id="accent-color" defaultValue="#ffffff" />
+                  <label htmlFor="accent-color" className="uppercase text-sm font-mono">
+                    Accent
+                  </label>
+                </div>
+              </div>
```

Comment:
I'm a bit worried with this re-order that color picking is going to be buried too deep since the "filter themes" section is so high and can even expand with more filter categories in the future.

maybe once "features" gets added search filter becomes more important but currently for me it seems not that obvious what to search for. would love to see someone unfamiliar with the project try to use search.

Reply:
Yeah it's a fair point. And it'd be good to do some user testing around search. I feel like color setting is an "advanced" or "optional" action in the side bar, and something you can do on the theme page too. While with a themes site theme list search/filtering is always gonna be at the top.

I polled Matt for an unbiased opinion and he landed on colors at the bottom with a similar "advanced" understanding. So I'm gonna merge this in, but I'm good with continuing to debate and test this post launch.